--- Bd. 21 - Folgen.tex before
+++ Bd. 21 - Folgen.tex after
@@ -2629,7 +2629,15 @@
 
 \section{Grenzwertregeln}
 
-\FormulaThmDeltaK[Lineare Grenzwertregeln]{%
+\FormulaThmDeltaKR[Lineare Grenzwertregeln]{%
+  \begin{aligned}[t]
+    &a,b\colon\mathbb N\to\mathbb R\dsep A,B,c\in\mathbb R\dsep
+      a_n\to A\dsep b_n\to B\vdash{}\\[-2pt]
+    &\text{(i)}\quad(a+b)_n\to A+B,\\[-2pt]
+    &\text{(ii)}\quad(a-b)_n\to A-B,\\[-2pt]
+    &\text{(iii)}\quad(ca)_n\to cA.
+  \end{aligned}%
+}{%
   a,b\colon\mathbb N\to\mathbb R\dsep
   A,B,c\in\mathbb R\dsep
   a_n\to A\dsep b_n\to B
@@ -3854,8 +3862,11 @@
 \closeproofpartwide
 
 \proofpartwideindR[Typisierung und Verschiedenheit der Marker]{%
-    c_0,c_1,c_2\in D\dsep c_0\neq c_1\land(c_0\neq c_2\land c_1\neq c_2)
-  }{c_0,c_1,c_2\in D\dsep c_0\neq c_1\land(c_0\neq c_2\land c_1\neq c_2)}[B21ReserveMarkerDistinct]
+  \begin{aligned}[t]
+    &\text{(i)}\quad c_0,c_1,c_2\in D,\\[-2pt]
+    &\text{(ii)}\quad c_0\neq c_1\land(c_0\neq c_2\land c_1\neq c_2).
+  \end{aligned}%
+}{c_0,c_1,c_2\in D\dsep c_0\neq c_1\land(c_0\neq c_2\land c_1\neq c_2)}[B21ReserveMarkerDistinct]
   \setcounter{proofstepnr}{30}% Fortlaufende Schritte dieses Beweises.
 \proofstepwidestar[]{%
       c_0\neq c_1\land(c_0\neq c_2\land c_1\neq c_2)}{%
